docs/conf.py

```diff
@@ -268,4 +268,7 @@ def setup(app):
 
 
 # Example configuration for intersphinx: refer to the Python standard library.
-intersphinx_mapping = {'http://docs.python.org/': None}
+intersphinx_mapping = {
+    'http://docs.python.org/': None,
```

Comment:
If we’re no longer leaving this as the default example, do we still need this line? I’m not sure I know enough about `intersphinx` to know what (if anything) it’s doing.